manager/state/raft/raft.go

```diff
@@ -394,6 +397,16 @@ func (n *Node) Run(ctx context.Context) error {
 				}
 			}
 
+			n.once.Do(func() {
```

Comment:
We need to process committed entries first on restart, after `RestartNode` and `Run`, otherwise `Campaign` panics. Thus the location of this portion.